modules/oc-mirror-creating-image-set-config.adoc

```diff
@@ -61,7 +61,7 @@ mirror:
 <2> Set the back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify `storageConfig` values, unless you are using the Technology Preview OCI feature.
 <3> Set the registry URL for the storage backend.
 <4> Set the channel to retrieve the {product-title} images from.
-<5> Add `graph: true` to generate the OpenShift Update Service (OSUS) graph image to allow for an improved cluster update experience when using the web console. For more information, see _About the OpenShift Update Service_.
+<5> Add `graph: true` to generate the `UpdateService` CR manifest that the OpenShift Update Service (OSUS) uses to provide a graph image for an improved cluster update experience when using the web console in a disconnected environment. For more information, see _About the OpenShift Update Service_.
```

Comment:
If setting `graph: true`, graph-data image that OSUS uses will be built and pushed to your mirror registry. In addition, the `UpdateService` CR manifest that the OpenShift Update Service (OSUS) uses will be generated as well. And it's not limited to web console only. It's easy for user to create OSUS instance using cli as well. 

cc @LalatenduMohanty @PratikMahajan for review. Thank you!

Reply:
agree with @shellyyang1989 ’s comment.